src/main/java/friday/Friday.java

```diff
@@ -0,0 +1,148 @@
+                    System.out.println(separator);
+                    break;
+                } else if (command.equalsIgnoreCase("list")) {
+                    System.out.println(separator);
+                    System.out.println("Affirmative, loading tasks...");
+                    for (int i = 0; i < tasks.size(); i++) {
+                        System.out.println(" " + (i + 1) + ". " + tasks.get(i));
+                    }
+                    System.out.println(separator);
+                } else if (command.startsWith("delete")) {
+                    String[] parts = Parser.splitOnce(command, "\\s+");
+                    if (parts.length < 2) {
+                        throw new FridayException("Apologies boss, that task number isn't recorded in my database.");
+                    }
+                    int index = Integer.parseInt(parts[1]) - 1;
+                    Task removed = tasks.remove(index);
+                    storage.save(tasks.asList());
+                    System.out.println(separator);
+                    System.out.println("Noted. I've removed this task:");
+                    System.out.println(" " + removed);
+                    System.out.println("Boss, you have " + tasks.size() + " tasks in the list.");
+                    System.out.println(separator);
+                } else if (command.startsWith("mark")) {
+                    String[] parts = Parser.splitOnce(command, "\\s+");
+                    if (parts.length < 2) {
+                        throw new FridayException("Apologies boss, that task number isn't recorded in my database.");
+                    }
+                    int index = Integer.parseInt(parts[1]) - 1;
+                    Task t = tasks.get(index);
+                    t.markAsDone();
+                    storage.save(tasks.asList());
+                    System.out.println(separator);
+                    System.out.println("Okay boss, I have marked this task as done:");
+                    System.out.println(" " + t);
+                    System.out.println(separator);
+                } else if (command.startsWith("unmark")) {
+                    String[] parts = Parser.splitOnce(command, "\\s+");
+                    if (parts.length < 2) {
+                        throw new FridayException("Apologies boss, that task number isn't recorded in my database.");
+                    }
+                    int index = Integer.parseInt(parts[1]) - 1;
+                    Task t = tasks.get(index);
+                    t.markAsNotDone();
+                    storage.save(tasks.asList());
+                    System.out.println(separator);
+                    System.out.println("Okay boss, I have marked this task as not done yet:");
+                    System.out.println(" " + t);
+                    System.out.println(separator);
+                } else if (command.startsWith("todo")) {
+                    String desc = command.length() > 4 ? command.substring(4).trim() : "";
```

Comment:
Nit: I think the code quality guidelines mention to avoid 'texting-style' spelling, would it be better to spell out description instead? Especially for consistency, since the full spelling `description` is also used in some other places.

```suggestion
                    String description = command.length() > 4 ? command.substring(4).trim() : "";
```

Reply:
I agree with this comment